vmdb/app/models/ems_openstack.rb

```diff
@@ -123,4 +123,8 @@ def vm_delete_evm_snapshot(vm, image_id)
     $log.debug err.backtrace.join("\n") if $log.debug?
     raise
   end
+
+  def self.event_monitor_class
```

Comment:
well before it was nil and there are checks for event_monitor_class.nil?, so basically the event_stop_monitor was never put in the queue.

I think I will just move it to another PR, it's possible someone removed it on purpose, or it's a bug and it's just missing. Would be good to dig a bit in the history of the code.